### PR TITLE
Add support for adding an issue when formatting the commit message

### PR DESCRIPTION
**Pull request by rwestberg**
Hi all,

Please review this change that allows setting the issue of a commit through the PR title.

Best regards,
Robin

Progress
--------
- [x] Change must not contain extraneous whitespace
- [x] Change must be properly reviewed

Approvers
---------
 * Erik Helin ([ehelin](@edvbld) - **Reviewer**)

### CODE REVIEW

File: host/src/main/java/org/openjdk/skara/host/github/GitHubPullRequest.java [hunk 1]
```diff
@@ -212,6 +212,11 @@ public String getTitle() {
         return json.get("title").asString();
     }
 
+    @Override
+    public void setTitle(String title) {
+        throw new RuntimeException("not implemented yet");
```
Comment (edvbld) [inline host/src/main/java/org/openjdk/skara/host/github/GitHubPullRequest.java]: Won't this be an issue if we run the unit tests as integration tests against real hosts?

File: host/src/main/java/org/openjdk/skara/host/gitlab/GitLabMergeRequest.java [hunk 2]
```diff
@@ -254,6 +254,11 @@ public String getTitle() {
         return json.get("title").asString();
     }
 
+    @Override
+    public void setTitle(String title) {
+        throw new RuntimeException("not implemented yet");
```
Comment (edvbld) [inline host/src/main/java/org/openjdk/skara/host/gitlab/GitLabMergeRequest.java]: Same issue here?

File: host/src/main/java/org/openjdk/skara/host/github/GitHubPullRequest.java (hunk 1 above)
Comment (rwestberg) [reply host/src/main/java/org/openjdk/skara/host/github/GitHubPullRequest.java]: You are quite right, I was planning to save it for another task, as there are already existing issues with the real host tests..

Comment (edvbld) [reply host/src/main/java/org/openjdk/skara/host/github/GitHubPullRequest.java]: Ah ok, lets tackle it later then :+1:

**Pull request closed without merging**